modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -159,6 +159,13 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`httpCompression`
+|`httpCompression` defines the policy for HTTP traffic compression. The optional `httpCompressionPolicy` field turns on compression for the MIME types, which are specified in the `RouterCompressionMIMETypes` variable. Compression is not enabled by default.
```

Comment:
`The optional `httpCompressionPolicy` `, i think 'The optional `ROUTER_ENABLE_COMPRESSION`